lang/en/package.html

```diff
@@ -0,0 +1,89 @@
+---
+layout: pages/detail
+scripts:
+- "/js/build/package.js"
+---
+{% comment %}
+this is the default package view to make the load percieve smoother,
+for the implementation, look at /js/lib/Details/index.js etc.
+{% endcomment %}
+<div id="pkg-detail">
+  <div data-reactroot="" class="details row">
+    <section class="details-main col-lg-8">
+      <header class="details-main--header">
+        <h2 class="details-main--title d-inline-block m-2">…</h2>
+        <div class="details-main--info d-inline-block m-2">
+          <a class="ais-Hit--ownerLink" href=""><img width="20" height="20" class="ais-Hit--ownerAvatar" src="https://res.cloudinary.com/hilnmyskv/image/fetch/w_40,h_40,f_auto,q_80,fl_lossy/https://avatars3.githubusercontent.com/u/22247014?v=3&amp;s=200">
+            <!-- react-text: 8 -->
+            <!-- /react-text -->
+          </a><span class="ais-Hit--popular null" title="0 downloads in the last 30 days">0</span>
+          <!-- react-empty: 10 --><span class="ais-Hit--version">0.0.0</span></div>
+        <p class="m-2 lead">…</p>
+        <!-- react-empty: 13 -->
+      </header>
+      <section id="readme" class="details-doc">
+        <h3 class="details-doc--title details-doc--title__readme"><a href="#readme">readme</a></h3>
+        <div>no readme found 😢</div>
```

Comment:
We see this message when the page takes time to load. Maybe a friendlier message could be nice? Something that tells the user that the data is loading?